src/main/java/string/Exer11.java

```diff
@@ -0,0 +1,17 @@
+package string;
+
+public class Exer11 {
+
+    public int subNumbers(String num1, String num2) {
+        return getNumber(num1) - getNumber(num2);
+    }
+
+    public int getNumber(String number) {
```

Comment:
Em phải tính toán trên String  vì số rất lớn

Reply:
:))
bắt cái đúng lỗi
hay qúa  :+1